static/src/javascripts/projects/commercial/modules/dfp/prepare-permutive.js

```diff
@@ -90,6 +95,18 @@ const generatePayload = (
     return cleanPayload;
 };
 
+const generatePermutiveIdentities = (
+    pageConfig: Config = {}
+): Array<PermutiveIdentity> => {
+    if (
+        typeof pageConfig.ophan === 'object' &&
+        typeof pageConfig.ophan.browserId === 'string'
```

Comment:
Could this ever be an empty string?

Reply:
Good shout, I agree if this happens it should be excluded.